tests-utils/src/main/java/tech/beshu/ror/utils/containers/ESWithReadonlyRestContainer.java

```diff
@@ -127,7 +127,7 @@ public static ESWithReadonlyRestContainer create(RorPluginGradleProjectJ project
               }
 
               builder.user("elasticsearch")
-                     .env("ES_JAVA_OPTS", "-Xms512m -Xmx512m -Djava.security.egd=file:/dev/./urandoms -Dcom.unboundid.ldap.sdk.debug.enabled=true")
+                     .env("ES_JAVA_OPTS", "-Xms512m -Xmx512m -Djava.security.egd=file:/dev/./urandoms -Dcom.unboundid.ldap.sdk.debug.enabled=true -Des.set.netty.runtime.available.processors=false")
```

Comment:
What is this for?

Reply:
This is related to: https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/issues/464. I've removed it from code, but it seems that we still need it tests, due to some issues with testcontainers (see: https://github.com/testcontainers/testcontainers-java/issues/1009)